tqdm/tests/tests_main.py

```diff
@@ -115,6 +116,31 @@ def test_comppath():
     rmtree(tmp, True)
 
 
+def test_autocomplete():
```

Comment:
Added a new test with the subset of options.
It's an extension of `test_comppath()`, if it looks fine, maybe we can club the two to remove some of the redundant code.